llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/SeedCollector.h

```diff
@@ -223,6 +223,11 @@ class SeedContainer {
     /// Note that the bundles themselves may have additional ordering, created
     /// by the subclasses by insertAt. The bundles themselves may also have used
     /// instructions.
+
+    // TODO: Range_size counts fully used-bundles. Further, iterating over
+    // anything other than the Bundles in a SeedContainer includes used seeds,
+    // so for now just check that removing all the seeds from a bundle also
```

Comment:
The sentence "so for now just check that removing all the seeds from a bundle also empties the bundle" seems to refer to the test.